source/code-generation/query-converter/bulk-conversion.txt

```diff
@@ -0,0 +1,93 @@
+.. _rm-bulk-conversion:
+
+===============
+Bulk Conversion
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can bulk convert one or more queries, stored procedures, triggers, 
+and views in bulk by enabling the multi-select toggle. Copy and paste 
+your queries into the query converter individually and then bulk convert
+a selection to update them to work with MongoDB and your migrated schema.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+- .. include:: /includes/fact-view-query-conversion-history.rst
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Navigate to the query converter pane
+
+      From the :guilabel:`Code Generation` tab, click the 
+      :guilabel:`Query Converter` pane.
+
+   .. step:: Open the SQL query converter view
+
+      - If it is your first time using the query converter in your 
+        project, click :guilabel:`Paste SQL Query`.
+      - If you have previously converted SQL code in your project, click the
+        :guilabel:`+ ADD` button on the left-hand pane next to 
+        :guilabel:`Queries`.
+
+   .. step:: Enable multi-selection
+
+      a. Click the :guilabel:`Enable multi-selection` toggle to show the query 
```

Comment:
(itemized list and a suggestion)
Globally, for a one item ordered list, you can probably just omit the list and make the text a paragraph. WDYT?

You might also explain what multi-selection is, and why you enable it?

Reply:
I refactored the first sentence to highlight the toggle functionality and followed-up with a general description of what bulk conversion does.